spec/lib/appsignal/hooks/http_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+describe Appsignal::Hooks::HttpHook do
+  before :context do
+    start_agent
+  end
+
+  if DependencyHelper.http_present?
+    describe "#dependencies_present?" do
+      subject { described_class.new.dependencies_present? }
+
+      it { is_expected.to be_truthy }
+    end
```

Comment:
```suggestion
    context "with instrument_http_rb set to true" do
      describe "#dependencies_present?" do
        subject { described_class.new.dependencies_present? }

        it { is_expected.to be_truthy }
      end
    end

    context "with instrument_http_rb set to false" do
      describe "#dependencies_present?" do
        before { Appsignal.config.config_hash[:instrument_http_rb] = false }
        after { Appsignal.config.config_hash[:instrument_http_rb] = true }
        subject { described_class.new.dependencies_present? }

        it { is_expected.to be_falsy }
      end
    end
```

Let's also test it if the config option is set to false.